engine/src/main/java/org/terasology/rendering/dag/tasks/SetWireframeTask.java

```diff
@@ -33,4 +33,9 @@ public SetWireframeTask(int mode) {
     public void execute() {
         GL11.glPolygonMode(GL_FRONT_AND_BACK, mode);
     }
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName() + ": " + mode;
```

Comment:
Let's not make this too hard on ourselves, let's put an IF statement here and let's generate:

`SetWireframeTask : enable wireframe` / `SetWireframeTask : disable wireframe`

in the log. We don't want to have to get to the opengl manual just to understand what is going on. =)